tests/test_inout.py

```diff
@@ -284,6 +284,34 @@ def test_complex_input_data(self):
         self.assertEqual(complex2.prop, 'data')
         self.assertEqual(complex.data, complex2.data)
 
+    def test_complex_input_data_with_binary_data_format(self):
+        complex = inout.inputs.ComplexInput(
+            identifier="complexinput",
+            title='MyComplex',
+            abstract='My complex input',
+            keywords=['kw1', 'kw2'],
+            workdir=self.tmp_dir,
+            supported_formats=[FORMATS.ZIP],
+            metadata=[Metadata("special data")],
+            default="/some/file/path",
+            default_type=SOURCE_TYPE.FILE,
+            translations={"fr-CA": {"title": "Mon input", "abstract": "Une description"}},
+            data_format=FORMATS.ZIP
+        )
+        complex.as_reference = False
+        complex.method = "GET"
+        complex.max_size = 1000
+        complex.data = b"some data"
+        # the data is enclosed by a CDATA tag in json
```

Comment:
this test is using base64? Not `CDATA`?

Reply:
Hello,

Yes this test using the Zip format that store data in base64 not using the CDATA patern.

Best regards

Reply:
👍